### PR TITLE
Solves #1 ✅

### LINKED ISSUE #1: Add Backend with go ⚡
- [x] go initialize
- [x] air for daemon
- [x] http package for server [fiber/Gorilla]

### PR DESCRIPTION
**Pull request by nimit2801**
Closes #1

### CODE REVIEW

**Comment by nimit2801** [comment]
Would request @shreyaparadkar to review and add any comments if necessary

File: server/tmp/build-errors.log [hunk 1]
```diff
@@ -0,0 +1 @@
+exit status 1exit status 1exit status 2exit status 2exit status 1exit status 1exit status 1exit status 1exit status 1exit status 1exit status 1exit status 1exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 2exit status 1
```
Comment (shreyaparadkar) [inline server/tmp/build-errors.log]: Remove tmp logs

Comment (nimit2801) [reply server/tmp/build-errors.log]: Thanks I've updated :)

**Review by shreyaparadkar — APPROVED** [review]
Looks good to me!

**Pull request merged**